CHANGELOG.md

```diff
@@ -0,0 +1,491 @@
+
+## Other Geyser Improvements
+
+Geyser.Label also allows you to use “nocolor” as the text color, which will then allow for coloring the text via CSS. It also picked up the Geyser.Label:rawEcho(txt) function, which skips over all the formatting assistance normally provided by Geyser.Label:echo().
+
+Geyser.MiniConsole gains Geyser.MiniConsole:display() which works just like regular display(), but outputs in the miniconsole instead of your main display.
+Tired of copying things to a miniconsole and not being able to click them? Wish you could keep your MXP links intact in a miniconsole? Well, now you can! Edru once again stepped up to the plate for this one.
+
+## New color functions for feedTriggers
+
+Have you struggled to test your color triggers? Tired of hand-crafting escape sequences when using feedTriggers()? Well, Demonnic was even if you weren’t, so he added some new functions for working with feedTriggers. decho2ansi() and hecho2ansi() will take any decho or hecho string and return an ansi colored string. The functions are used by the new feedTriggers variants, dfeedTriggers and hfeedTriggers which allow for you to send colored text to the trigger engine for testing your color triggers. Also added is cfeedTriggers, which uses color names but uses a limited color table which corresponds to the ansi standard colors, IE black, red, green, yellow, blue, magenta, cyan, white, and lightBlack, lightRed, etc. You can also use a number from 0-255 to get the ansi 256 color equivalent.
+
+## Multi-view forever!
+
+Now you can open ALL THE PROFILES! and move between them with your mouse and keyboard shortcuts without it turning multiview off. Have fun multiplaying!
+
+![](https://www.mudlet.org/wp-content/uploads/2020/11/Selection_368-1024x446.png)
+
+## Telnet CHARSET support
+
+Game admins will be happy about this feature – it is now possible to automatically set Mudlet’s text encoding – for example, to be utf8 in case your game supports emojis – using the [CHARSET protocol](https://tools.ietf.org/html/rfc2066). Big thanks to Tamarindo from [StickMUD](https://www.stickmud.com/) for pulling that one in.
+
+## Music/sound in MXP
+
+Tamarindo also went ahead and added sound/music support to Mudlet’s MXP implementation. That said, there’s a much more modern [Mud Client Media Protocol](https://wiki.mudlet.org/w/Special:MyLanguage/Standards:MUD_Client_Media_Protocol) that you should aim to use 😉
+
+## Scrollbar improvements
+
+You’ll not be surprised to hear this one is Edru as well. Had you noticed Mudlet’s scrollbars behaving a bit… oddly? Jumping around, not resizing itself, stuff like that? Well, Edru’s put a lot of work into this and it should all work much more smoothly now.
+ 
+
+## macOS 10.13 High Sierra
+
+Apple will stop supporting 10.13 High Sierra in November 2020, and Mudlet will have to follow suit. Thus Mudlet 4.10 will be the last version to support 10.13, which had a good run of 3 years!
```

Comment:
Needs edit now doesn't it?
```suggestion
Apple has stopped supporting 10.13 High Sierra since November 2020, and Mudlet had to follow suit. Thus Mudlet 4.10 was the last version to support 10.13, which had a good run of 3 years!
```

Reply:
This was a verbatim copy of the website release news post.
We never go back to update those at all, at all.
Isn't it funny how easy this now becomes?
For CHANGELOG, keeping it current may make sense indeed.